packages/url/package.json

```diff
@@ -14,10 +22,16 @@
     "url": "https://github.com/contentful/field-editors"
   },
   "scripts": {
-    "watch": "tsdx watch",
-    "build": "tsdx build",
-    "test": "tsdx test --env=jsdom --watch",
-    "test:ci": "tsdx test --env=jsdom --ci",
+    "watch": "yarn concurrently \"yarn:watch:*\"",
+    "watch:cjs": "yarn build:cjs -w",
+    "watch:esm": "yarn build:esm -w",
+    "watch:types": "yarn build:types --watch",
+    "build": "yarn build:types && yarn build:cjs && yarn build:esm",
+    "build:types": "tsc --outDir dist/types --emitDeclarationOnly",
+    "build:cjs": "swc src -d dist/cjs -C module.type=commonjs",
+    "build:esm": "swc src -d dist/esm",
+    "test": "jest --env=jsdom --watch",
```

Comment:
as we set the env in test and test:ci and in every project, speaks something against moving this into the jest.config (or base jest config)